tools/libxl/libxl_arm.c

```diff
@@ -952,7 +985,7 @@ static int libxl__prepare_dtb(libxl__gc *gc, libxl_domain_build_info *info,
         FDT( make_hypervisor_node(gc, fdt, vers) );
 
         if (pfdt) {
-            FDT( copy_partial_fdt(gc, fdt, pfdt) );
+            FDT( copy_partial_fdt(gc, fdt, pfdt, info) );
             FDT( copy_coprocs_nodes(gc, fdt, pfdt, info) );
```

Comment:
So this patchset is based on "coprocs"? Shouldn't it be based on xen.org master?